Packs/Remedy-On-Demand/Integrations/RemedyOnDemand/RemedyOnDemand.yml

```diff
@@ -255,7 +248,10 @@ script:
       description: Ticket urgency
     - contextPath: Ticket.Type
       description: Ticket service type
-    description: Update exiting incident
   isfetch: true
-hidden: false
+  runonce: false
+  script: ''
+  type: javascript
 fromversion: 5.0.0
+tests:
+- No tests (auto formatted)
```

Comment:
This was submitted through the XSOAR UI. Did not do any command line tools. This is also why so many fields were renamed to use underscores instead of dashes - it was forced by the UI